doc/ocp_lock/lock_spec.ocp

```diff
@@ -278,6 +278,8 @@ To mitigate against cryptographic attacks on the HPKE keypair that rely on repea
 
 ![MPK generation](./diagrams/mpk_generate.drawio.svg){#fig:mpk-generation}
 
+To comply with SP 800-38D [@{sp800-38d}] section 8.3's stipulation that AES-GCM-Encrypt may only be invoked at most 2^32^ times with a given key, drive firmware is responsible for ensuring that no more than 2^32^ MPKs are generated for a given SEK.
```

Comment:
This stipulation is true for a random IV. If we used a deterministic construction, we might be able to avoid this, but we'd then have to keep track of a counter in the IV.

Another potential alternative could be, instead of ever using an MPK directly, you always derive a new key from it (using a counter), so that you are never using the same key twice. But I understand that complicates these flows even further.